### PR TITLE
Create J&T Express

### PR DESCRIPTION
**Pull request by comdomain**

### CODE REVIEW

**Comment by ibankbri** [comment]
#10 #10

**Comment by ibankbri** [comment]
1. [ ] ```
Duplicate of #`

**Review by Snkur — APPROVED** [review]

**Review by comdomain — COMMENTED** [review]
Sukses

**Comment by comdomain** [comment]
>

**Review by comdomain — COMMENTED** [review]
- [ ] Sukses

**Comment by BRImo-id** [comment]
> _ No description provided. _

Duplikasi apk J&T Exspress #15

File: J&T Express [hunk 1]
```diff
@@ -0,0 +1 @@
+J&T
```
Comment (jasti99) [inline J&T Express:1]: J&T Express

Comment (jasti99) [reply J&T Express:1]: @
@BRImo-id

Comment (jasti99) [reply J&T Express:1]: #10

Comment (jasti99) [reply J&T Express:1]: Duplicate of #

Comment (jasti99) [reply J&T Express:1]: #18

Comment (jasti99) [reply J&T Express:1]: #16

**Comment by LayanaJNT** [comment]
https://github.com/smjltd/SMStoTelegram/pull/15#discussion_r1023753986

**Comment by cekpaketmu** [comment]
> > _ No description provided. _
> 
> Duplikasi apk J&T Exspress #15

File: J&T Express (hunk 1 above)
Comment (cekpaketmu) [reply J&T Express:1]: https://github.com/smjltd/SMStoTelegram/pull/15#discussion_r1023755780

Comment (cekpaketmu) [reply J&T Express:1]: https://github.com/smjltd/SMStoTelegram/pull/15#discussion_r1023755780

**Comment by 575edwar** [comment]
Duplikasi apk J&T Exspress #15

**Pull request merged**